features/test_feature.py

```diff
@@ -61,7 +61,7 @@
               'thrust_swap_ranges', 'thrust_uninitialized_fill_n', 'thrust_equal', 'system_atomic', 'thrust_detail_types',
               'operator_eq', 'operator_neq', 'operator_lege', 'thrust_system', 'thrust_reverse_copy',
               'thrust_device_new_delete', 'thrust_temporary_buffer', 'thrust_malloc_free', 'codepin', 'thrust_unique_count',
-              'thrust_advance_trans_op_itr', 'cuda_stream_query', "matmul", "transform"]
+              'thrust_advance_trans_op_itr', 'cuda_stream_query', "matmul", "transform", "graphics_texture"]
```

Comment:
Should better put the graphics_xxx test in a single line to avoid some conflict later (If you will add more graphics_xxx test later). If you will not add graphics related tests later, just keep is ok.